databook/_toc.yml

```diff
@@ -16,74 +16,95 @@ chapters:
-      title: Ophys Experiment Data
-    - file: visual-behavior/VBO-Tutorial-Compare_trial_types
-      title: Tutorial Comparing Trial Types
-    - file: visual-behavior/VBO-Tutorial-Downloading_data_and_exploring_the_manifest
-      title: Tutorial Downloading Data
-  - file: visual-behavior/VB-Neuropixels
-    title: Visual Behavior Neuropixels
+    - file: physiology/ephys/visual-coding-np/vcnp-quickstart
+    - file: physiology/ephys/visual-coding-np/vcnp-data-access
+    - file: physiology/ephys/visual-coding-np/vcnp-units
+    - file: physiology/ephys/visual-coding-np/vcnp-receptive-fields
+    - file: physiology/ephys/visual-coding-np/vcnp-lfp
+    - file: physiology/ephys/visual-coding-np/vcnp-optotagging
+    - file: physiology/ephys/visual-coding-np/vcnp-session
+    - file: physiology/ephys/visual-coding-np/vcnp-cheatsheet
+    - file: physiology/ephys/visual-coding-np/vcnp-links
+  - file: physiology/ephys/np-ultra/np-ultra-background                     # Also a placeholder that needs actual content
+  - file: physiology/cell-type-lookup-table/ctlut-background
     sections:
-    - file: visual-behavior/VBN-Dataset
-    - file: visual-behavior/VBN-SessionData
-    - file: visual-behavior/VBN-Tutorial-Aligning_Neural_Data_to_Stimuli.ipynb
-      title: Tutorial Aligning Neural Data to Stimuli
-    - file: visual-behavior/VBN-Tutorial-Aligning_Behavior_Data_to_Task_Events.ipynb
-      title: Tutorial Aligning Behavioral Data to Task Events
-    - file: visual-behavior/VBN-Tutorial-Analyzing_LFP_Data.ipynb
-      title: Tutorial Analyzing LFP data
-- file: microns-em/em-background
+    - file: physiology/cell-type-lookup-table/ctlut-session-data
+    - file: physiology/cell-type-lookup-table/ctlut-optotagging
+    - file: physiology/cell-type-lookup-table/ctlut-accessing-data
+    - file: physiology/cell-type-lookup-table/ctlut-identifying-tagged-units
+  # New files here
+- file: stimuli/stimuli
   sections:
-  - file: microns-em/em-structural-data.md
-  - file: microns-em/proofreading.md
-  - file: microns-em/em-neuroglancer.md
-  - file: microns-em/em-dash-apps.md
-  - file: microns-em/em-caveclient-setup.md
-  - file: microns-em/em-querying-tables.md
-  - file: microns-em/key-tables.md
-  - file: microns-em/em-volume-data.md
-  - file: microns-em/em-skeletons.md
-  - file: microns-em/em-meshes.md
-  - file: microns-em/em-functional_data.md
-  - file: microns-em/em-coordinates.md
-  - file: microns-em/em-ultrastructure.md
+  - file: stimuli/passive-visual-stimuli
+  - file: stimuli/change-detection-task
+  - file: stimuli/visual-behavior/vb-background                 # This section needs to go somewhere, but not sure where yet; background stuff on visual behavior.
+    sections:
+    - file: stimuli/visual-behavior/VB-Behavior
+      title: Visual Behavior Task
+      sections:
+      - file: stimuli/visual-behavior/VB-BehaviorSessionData
+        title: Behavior Session Data
+    - file: stimuli/visual-behavior/VB-Ophys
```

Comment:
VB-Ophys, VBO-Dataset, VBO-ExperimentData, VBO_Tutorial... (X2)
VB-Neuropixels, VBN-etc
these all need to be in the physiology section, not the stimuli section. The only one that should be in the stimulus section is VB-Behavior.